code/modules/surgery/organs/external/tails.dm

```diff
@@ -149,18 +149,16 @@
 		paired_spines = null
 
 /obj/item/organ/external/tail/lizard/start_wag()
-	. = ..()
-
 	if(paired_spines)
 		var/datum/bodypart_overlay/mutant/spines/accessory = paired_spines.bodypart_overlay
 		accessory.wagging = TRUE
-
-/obj/item/organ/external/tail/lizard/stop_wag()
 	. = ..()
 
+/obj/item/organ/external/tail/lizard/stop_wag()
 	if(paired_spines)
 		var/datum/bodypart_overlay/mutant/spines/accessory = paired_spines.bodypart_overlay
 		accessory.wagging = FALSE
+	. = ..()
```

Comment:
```suggestion
	return ..()
```

there's no codewise difference between this but this is a bit more explicit since it's at the end of the proc